test/Microsoft.TestPlatform.TestHostProvider.UnitTests/Hosting/TestHostManagerCallbacksTests.cs

```diff
@@ -94,4 +98,103 @@ public void ErrorReceivedCallbackShouldAppendNewLineApproprioritlyWhenReachingMa
 
         Assert.AreEqual("1234" + Environment.NewLine.Substring(0, 1), _testHostProcessStdError.ToString());
     }
+
+    [TestMethod]
+    public void ErrorReceivedCallbackShouldNotCrashIfInvalidProcessHandle()
+    {
+        var handleProcessHelper = new InvalidHandleProcessHelper();
```

Comment:
I believe you could simply use a mock here.

Reply:
yeah....I start to hate mocks...fakes are so simple  will do.